roles/template_service_broker/tasks/install.yml

```diff
@@ -68,10 +68,14 @@
     {{ openshift_client_binary }} process --config={{ mktemp.stdout }}/admin.kubeconfig -f "{{ mktemp.stdout }}/{{ __tsb_rbac_file }}"
     | {{ openshift_client_binary }} auth reconcile --config={{ mktemp.stdout }}/admin.kubeconfig -f -
 
+# TSB Route
+- shell: oc get routes -n kube-service-catalog -o jsonpath='{.items[*].spec.host}{"\n"}'
```

Comment:
Should use `{{ openshift_client_binary }}` to work on containerized deployments

Reply:
Agreed, Atomic deploys `oc` to `/usr/local/bin/` (`/var/usrlocal/bin/`) which is in the sshd default path but not in the sudoers `secure_path` (affecting deployments when `ansible_ssh_user` is not `root` in an advanced install).

Reply:
is it valid namespace here?
in original it was 'openshift-template-service-broker' vs 'kube-service-catalog'
is it the same apiserver?